PRODUCTION.md

```diff
@@ -0,0 +1,55 @@
+# Scality S3server in production with Docker
+
+## Using Docker Volume
+
+in production
+
+S3Server runs with a file backend by default.
+
+So, by default, the data are stored inside your S3server Docker container.
+
+However, you **MUST** use Docker volumes to host the data and the metadata of
```

Comment:
However, if you want your data and metadata to persist, you **MUST** use Docker volumes to host your data and metadata outside your s3server Docker container.  Otherwise, the data and metadata will be destroyed when you erase the container.